test/verify/check-connection

```diff
@@ -857,6 +857,7 @@ class TestConnection(testlib.MachineCase):
         self.assertIn("HTTP/1.1 404 Not Found\r\n", headers)
 
     @testlib.skipWsContainer("ssh root login not allowed")
+    @testlib.skipBrowser("Chromium 133 does not survive large downloads", "chromium")
```

Comment:
Could this grow a link to your chromium bug? You'll need another push anyway to mop up the new pixel refs. Thanks!

Reply:
Yes!